server/server.go

```diff
@@ -332,6 +332,33 @@ func copyHeader(srcHeaders http.Header, dstHeaders http.Header) {
 	}
 }
 
+// readClusterIDsForOrgID reads the list of clusters for a given
+// organization from aggregator
+func (server HTTPServer) readClusterIDsForOrgID(orgID types.OrgID) ([]types.ClusterName, error) {
+	aggregatorURL := httputils.MakeURLToEndpoint(
+		server.ServicesConfig.AggregatorBaseEndpoint,
+		ira_server.ClustersForOrganizationEndpoint,
+		orgID,
+	)
+
+	response, err := http.Get(aggregatorURL)
+	if err != nil {
+		return nil, err
+	}
+
+	type clustersResponse struct {
+		Status   string              `json:"status"`
+		Clusters []types.ClusterName `json:"clusters"`
+	}
+
+	var recvMsg clustersResponse
```

Comment:
btw, you could omit type declaration here 
```
var recvMsg struct {
...
```

Reply:
TYVM, I will fix it